### ISSUE TITLE
Dropbox mock

### ISSUE THREAD

**Issue by rizzatti** [rizzatti/iosCourse#1]
@timothy1ee
@codepathreview

**Comment by timothy1ee** [comment]
Your settings screen looks squashed, and it doesn't scroll?

**Comment by rizzatti** [comment]
@timothy1ee Yes, I need to improve that. I followed your advice from the video and after a while stuck at that problem, I moved on to keep to ball rolling. I'll look closely into how scrollable areas work later today.

**Comment by timothy1ee** [comment]
OK, the major learnings from last week were general usage of Xcode, how to set up navigation and hit states, and how to work with scrollable regions. Can you finish that last item today? It's crucial to get that skill before moving on to the next set of skills.
